quartz/plugins/transformers/ofm.ts

```diff
@@ -294,6 +298,18 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
             ])
           }
 
+          if (opts.parseArrows) {
+            replacements.push([
+              arrowRegex,
+              (_value: string, ..._capture: string[]) => {
+                return {
+                  type: "html",
+                  value: `<span>&#10230;</span>`,
```

Comment:
why ⟶? let's go back to `&rarrow;`

Reply:
also hm maybe other arrow support later but up to you whether you want to implement it

Reply:
Hmm it seems like `&rarrow;` is not renderred correctly 🤔 Not sure if there is something I'm doing wrong or sth.

Reply:
let's support different arrow later? Since obsidian only render `rarr` and `xrarr`.